uber/models.py

```diff
@@ -1339,7 +1339,7 @@ def is_unassigned(self):
 
     @property
     def is_dealer(self):
-        return self.ribbon == c.DEALER_RIBBON or self.badge_type == c.PSEUDO_DEALER_BADGE
+        return self.ribbon == c.DEALER_RIBBON or self.badge_type == c.PSEUDO_DEALER_BADGE or self.group and self.group.is_dealer and self.paid == c.PAID_BY_GROUP
```

Comment:
We shouldn't need those parentheses -- `and` binds tighter than `or` -- but I can include them for better legibility. I'll break the line up a bit as well. Good point. :)